### DISCUSSION TITLE
taxonomy for BLAST databases

### DISCUSSION THREAD

**Discussion by MadisMetsis** [bbuchfink/diamond#451] [Q&A]
I have used diamond some years ago and came back as was delighted to see that BLAST database is possible to use in 2.0.8. However, reading the manual I cannot firn out how the taxonomy is handled in the case of BLASR DB usage. both in case of --taxonlist <list> in input and staxids>>sphylums in format 6 output. In an earlier version, the issue was fixed during database formatting.

YS
MM

**Comment by bbuchfink** [comment]
Taxonomy features are not yet supported for blast databases, you will have to use the regular db format for that, at least for now.

**Reply by Proginski** [reply]
Is it still the case ? Because downloading a non corrupted nr fasta can be challenging (till now 3 attempts on 3 devices : 3 fails).. ::))

**Reply by bbuchfink** [reply]
Yes this is still the case. I also recommend `update_blastdb.pl` to download the databases, then convert to FASTA.

**Comment by MadisMetsis** [comment]
I would suggest downloading formatted blast nr db - it is in multiple
pieces and therefore easier. And then convert it back into FASTA.
YS
MM

Kontakt Paul Roginski ***@***.***>) kirjutas kuupäeval T,
22. märts 2022 kell 13:37:

> Is it still the case ? Because downloading a non corrupted nr fasta can be
> challenging (till now 3 attempts on 3 devices : 3 fails).. ::))
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/bbuchfink/diamond/discussions/451#discussioncomment-2414969>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATT764B36DGMHZMH6SZQEV3VBHEITANCNFSM42XPH62Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
I have had this world famous COVID and survived, its was really severe. I
also had two shots of vaccine. From now on I will talk about on those
issues only when I am paid.

**Reply by Proginski** [reply]
Finally changing "ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz" to "http://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz" did the job !
